extra/command-not-found.zsh

```diff
@@ -5,6 +5,15 @@ command_not_found_handler() {
   if [[ -n "$pkgs" ]]; then
     printf '%s may be found in the following packages:\n' "$cmd"
     printf '  %s\n' $pkgs[@]
+    if [[ -n $PKGFILE_PROMPT_INSTALL_MISSING ]]; then
```

Comment:
I don't think we should elide the check here that's done in bash -- invoking this behavior only when pkgfile returns a single result. At best, you offer to install a randomly selected package that might not be what the user wants.